src/main/java/alvioneurope/gsergeev/javagroup/clientlogimporter/ExecutedCommand.java

```diff
@@ -11,5 +12,65 @@
     private String operationId;
     private int responseLength;
     private int execTime;
+    private boolean isExecutedCommand;
```

Comment:
Please use a naming convention for boolean 
http://stackoverflow.com/questions/5322648/for-a-boolean-field-what-is-the-naming-convention-for-its-getter-setter
Plese rename getter also